app/src/test/java/com/example/newsapp/datasource/MeLocalDataSourceTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.example.newsapp.datasource
+
+import com.example.newsapp.data.DummyData
+import com.example.newsapp.data.local.MeLocalDataSource
+import com.example.newsapp.data.local.room.NewsDao
+import com.example.newsapp.data.local.room.ProfileDao
+import com.example.newsapp.modules.meLocalSourceTestModule
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.koin.test.KoinTest
+import org.koin.test.KoinTestRule
+import org.koin.test.inject
+import org.mockito.Mockito
+import kotlin.test.assertTrue
+
+@ExperimentalCoroutinesApi
+@RunWith(JUnit4::class)
+class MeLocalDataSourceTest : KoinTest {
+
+    private val newsDao = mock<NewsDao>()
+    private val profileDao = mock<ProfileDao>()
+    private val dataSource: MeLocalDataSource by inject()
+    private val expectedProfile = DummyData.PROFILE_ENTITY
+    private val expectedNews = listOf(DummyData.NEWS_ENTITY)
+
+    @get:Rule
+    val koinTestRule = KoinTestRule.create {
+        printLogger()
+        modules(meLocalSourceTestModule(newsDao, profileDao))
+    }
+
+    @Test
+    fun `insert profile`() = runBlockingTest {
```

Comment:
Please provide more clear test name by including the expected scenario eg, insert profile success, insert profile failed, etc